secantTests/HomeTests/HomeTests.swift

```diff
@@ -209,13 +214,52 @@ class HomeTests: XCTestCase {
         testScheduler.advance(by: 0.01)
         
         // expected side effects as a result of .onAppear registration
+        store.receive(.updateRoute(nil))
         store.receive(.synchronizerStateChanged(.unknown))
         store.receive(.updateSynchronizerStatus)
 
         // long-living (cancelable) effects need to be properly canceled.
         // the .onDisappear action cancles the observer of the synchronizer status change.
         store.send(.onDisappear)
     }
+
+    func testOnAppear_notEnoughSpaceOnDisk() throws {
```

Comment:
Do we miss testOnAppear_enoughSpaceOnDisk? It should go the other way and register for the synchronizer's status update and spin up the whole wallet.

Reply:
`testOnAppear` does exactly this. It was already there and I just mocked empty disk for it.